governance-and-automation/permissions-review/create-apply-access-review-policies.md

```diff
@@ -133,9 +133,19 @@ The warning serves as a reminder that **if you have rules set up**, manually cha
 
 {% endhint %}
 
-
 ![Apply Policy Dialog - Warning](../../.gitbook/assets/create-access-review-policies-apply-policy-warning.png)
 
+{% hint style="warning" %}
+**Please note!** 
+
+Only **one Access Review policy can be active at a time** on a workspace.
+When a new Access Review policy is created and applied to workspaces, **it will be triggered depending on the date set in the Settings of the Access Review Option**.
+ * **If the date** in the Settings for the Access Review Option **has not changed**, it will be triggered on the set date in the future and then occur depending on the recurrence date set for the individual policy.
+ * **If the date** in the Settings for the Access Review Option **is set in the future**, it will be triggered on the set date in the future and then occur depending on the recurrence date set for the individual policy.
+ * **If the date** in the Settings for the Access Review Option **is set in the past**, the policy will be triggered based on the recurrence set for the policy.
+
```

Comment:
mislim da bi bilo jasnije da bude Settings > Access Review Options
npr.: 
 * **If the date** in Settings > Access Review Options **has not changed**, the newly applied Access Review will be triggered on the set date in the future.
kod svih točaka bih stavio primjer, jer meni i dalje tu nije jasno kada će se nešto triggerati
recurrence date ne postoji - postoji samo recurrence koji se može definirati by policy da bude min 1 mjesec pa do 12 mjeseci, sa 1 month inkrementom